galasa-extensions-parent/dev.galasa.extensions.common/src/main/java/dev/galasa/extensions/common/couchdb/CouchdbStore.java

```diff
@@ -143,15 +145,15 @@ protected List<ViewRow> getAllDocsFromDatabase(String dbName) throws CouchdbExce
      * @throws CouchdbException             if there was a problem accessing the
      *                                      CouchDB store or its response
      */
-    protected List<ViewRow> getAllDocsByLoginId(String dbName, String loginId) throws CouchdbException {
+    protected List<ViewRow> getAllDocsByLoginId(String dbName, String loginId, String viewName) throws CouchdbException {
```

Comment:
Since this gets documents by login ID, shouldn't this be a method in the auth store rather than the base couchdb store?